services/api/src/app.py

```diff
@@ -113,6 +114,8 @@ async def request(
             .save(db_connection)
         )
 
+    _save_request(request)
```

Comment:
A few things:

Might want to add RequestModel to the /schema module (wrap the nnsight RequestModel in a new NDIF one to add saving/logging logic) 

Also need to try and save the request in every possible point where there can be an error ( api, request, model)